packages/blueprints-v2-addon/update-blueprints.mjs

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 'use strict';
 
+// eslint-disable-next-line import-x/default
```

Comment:
Unsure yet if this is a false positive. https://github.com/kellyselden/git-diff-apply/compare/v7.0.0...v7.0.4 seemed to show only formatting changes.